grafana: explained metrics in log backup row (#17387)
pingcap/docs: grafana-tikv-dashboard.md

- Encrypt/decrypt data nanos: The histogram of duration on encrypting/decrypting data each time
- Read/write encryption meta duration: The time consumed for reading/writing encryption meta files

### Log Backup

- Handle Event Rate: The speed of handling write events
- Initial Scan Generate Event Throughput: Incremental scanning speed when generating a new listener stream

> - Abnormal Checkpoint TS Lag: Lag of the current checkpoint TS to the present time for each task

Reviewer: ```suggestion
- Abnormal Checkpoint TS Lag: The lag of the current checkpoint TS to the present time for each task
```

- Memory Of Events: An estimated amount of memory occupied by temporary data generated by incremental scanning
- Observed Region Count: The number of Regions currently listened to
- Errors: The number and type of retryable and non-fatal errors
- Fatal Errors: The number and type of fatal errors. Usually fatal errors cause the task to be paused.
- Checkpoint TS of Tasks: Checkpoint TS for each task
- Flush Duration: The heat map of how long it takes for moving cached data to external storage
- Initial Scanning Duration: The heat map of how long it takes for incremental scanning when creating a new listening stream
- Convert Raft Event Duration: The heat map of how long it takes to transform a Raft log entry into backup data after creating a listening stream
- Command Batch Size: The batch size (within a single Raft group) of the listening Raft command
- Save to Temp File Duration: The heat map of how long it takes to temporarily store a batch of backup data (spanning several Tasks) into the temporary file area

> - Write to Temp File Duration: The heat map of how long it takes to temporarily store a batch of backup data (from a particular task) into the temporary files area

Reviewer: ```suggestion
- Write to Temp File Duration: The heat map of how long it takes to temporarily store a batch of backup data (from a particular task) into the temporary file area
```

- System Write Call Duration: The heat map of how long it takes to write a batch of backup data (from a Region) to a temporary file
- Internal Message Type: The type of messages received by the actor responsible for the log backup within TiKV
- Internal Message Handling Duration (P90|P99): The speed of consuming and processing each type of messages
- Initial Scan RocksDB Throughput: The read traffic generated by RocksDB internal logging during incremental scanning
- Initial Scan RocksDB Operation: The number of individual operations logged internally by RocksDB during incremental scanning
- Initial Scanning Trigger Reason: The reason for triggering incremental scanning
- Region Checkpoint Key Putting: The number of checkpoint operations logged to the PD

> **Note:**
>

> > The following monitoring items all use TiDB as their data source, but they have some impact on the log backup process. For ease of reference, they are placed in TiKV Details. In addition, TiKV actively pushes progress most of the time, so it is normal for some of the following monitoring items to occasionally not have sampled data.

Reviewer: ```suggestion
> The following monitoring metrics all use TiDB nodes as their data source, but they have some impact on the log backup process. Therefore, they are placed in the **TiKV Details** dashboard for ease of reference. TiKV actively pushes progress most of the time, but it is normal for some of the following monitoring metrics to occasionally not have sampled data.
```

- Request Checkpoint Batch Size: The request batch size when the log backup coordinator requests checkpoint information for each TiKV

> - Tick Duration \[P99|P90\]:  The time taken by the tick inside the coordinator

Reviewer: ```suggestion
- Tick Duration \[P99|P90\]: The time taken by the tick inside the coordinator
```

> - Region Checkpoint Failure Reason: A record of the reason why a Region checkpoint cannot advance within the coordinator

Reviewer: ```suggestion
- Region Checkpoint Failure Reason: The reason why a Region checkpoint cannot advance within the coordinator
```
